packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -40,72 +41,87 @@ const StyledList = styled.ul`
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
     grid-column-gap: ${GEL_SPACING_DBL};
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(
-      ${({ children }) => getRowCount(children, 3)},
-      auto
+    grid-template-columns: repeat(
+      ${({ trustProjectLink }) => (trustProjectLink ? 4 : 3)},
+      1fr
     );
+    grid-template-rows: repeat(${({ links }) => getRowCount(links, 3)}, auto);
     column-count: 3;
   }
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
     grid-column-gap: ${GEL_SPACING_DBL};
     grid-template-columns: repeat(4, 1fr);
-    grid-template-rows: repeat(
-      ${({ children }) => getRowCount(children, 4)},
-      auto
-    );
+    grid-template-rows: repeat(${({ links }) => getRowCount(links, 4)}, auto);
     column-count: 3;
   }
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     grid-column-gap: ${GEL_SPACING_DBL};
-    grid-template-columns: repeat(5, 1fr);
-    grid-template-rows: repeat(
-      ${({ children }) => getRowCount(children, 5)},
-      auto
+    grid-template-columns: repeat(
+      ${({ trustProjectLink }) => (trustProjectLink ? 5 : 4)},
+      1fr
     );
-    column-count: 3;
+    grid-template-rows: repeat(${({ links }) => getRowCount(links, 5)}, auto);
+    column-count: 4;
   }
-  > li:first-child {
-    border-bottom: 1px solid ${C_SHADOW};
+  ${({ trustProjectLink }) =>
+    trustProjectLink &&
+    `> li:first-child {
+    border-bottom: 0.0625rem solid ${C_SHADOW};
     padding: ${GEL_SPACING} 0;
     margin-bottom: ${GEL_SPACING};
     grid-column: 1/-1;
     width: 100%;
     column-span: all;
-  }
+  }`}
 `;
 
 const StyledListItem = styled.li`
   min-width: 50%;
   column-gap: 1rem;
+  break-inside: avoid-column;
 `;
 
-const List = ({ links }) => (
-  <StyledList role="list">
-    {links.map(link => (
-      <StyledListItem key={link.text} role="listitem">
-        <Link text={link.text} href={link.href} />
-      </StyledListItem>
-    ))}
-  </StyledList>
+const List = ({ links, trustProjectLink }) => (
+  <>
+    <StyledList role="list" trustProjectLink={trustProjectLink} links={links}>
+      {trustProjectLink && (
+        <StyledListItem key={trustProjectLink.text} role="listitem">
+          <Link text={trustProjectLink.text} href={trustProjectLink.href} />
+        </StyledListItem>
+      )}
+      {links.map(link => (
+        <StyledListItem key={link.text} role="listitem">
+          <Link text={link.text} href={link.href} />
+        </StyledListItem>
+      ))}
+    </StyledList>
+  </>
 );
 
 const linkPropTypes = shape({
   href: string.isRequired,
   text: string.isRequired,
 });
 
+const trustProjectLinkPropTypes = shape({
+  href: string,
+  text: string,
+});
+
 List.propTypes = {
   links: arrayOf(linkPropTypes.isRequired).isRequired,
+  trustProjectLink: oneOfType([trustProjectLinkPropTypes, bool]),
```

Comment:
I don't think you need to be passing in a link of type boolean. This should be sufficient. Then you don't need to define `trustProjectLinkPropTypes`. 
```suggestion
  trustProjectLink: linkPropTypes,
```

Reply:
resolved